index.d.ts

```diff
@@ -0,0 +1,37 @@
+// By: Yusuf Can INCE <ycanince@gmail.com>
```

Comment:
May I suggest you to add  on top this line

// SPDX-Licence-Indentifier: MIT

for The MIT License (MIT)

By can be also replaced by 

Copyright: ... 2020+